python/res/job_queue/workflow_runner.py

```diff
@@ -2,6 +2,20 @@
 from res.job_queue import Workflow
 from res.util.substitution_list import SubstitutionList
 
+class WorkflowRunnerThread(Thread):
+    def __init__(self, target, name):
+        super(WorkflowRunnerThread, self).__init__(target=target, name=name)
+        self.setDaemon(True)
+        self._exception = None
+        # in python2.7 it's called __target therefore we need to inject this too
```

Comment:
I don't understand this comment.

Reply:
The difference of Python2.7 vs Python3.3+ is `thread.__target` vs `thread._target`; therefore we set an attribute `thread._target` (which is tackled by __init__ for Python3.3+)